### ISSUE TITLE
error 500 with phpbb 3.2.2

### ISSUE THREAD

**Issue by lhbt** [brunoais/phpbbextf_read_other#11]
[15-Apr-2018 20:44:18 Europe/London] PHP Parse error:  syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in /home/****/public_html/forum/ext/brunoais/readOthersTopics/event/main_listener.php on line 334

php version: 5.6.35

hope that helps! would really love to see the extension working with 3.2.2.
Cheers

**Comment by brunoais** [comment]
https://github.com/brunoais/readOthersTopics/blob/1.0.1/event/main_listener.php#L334
Are you sure?
I see no issue with the PHP code. It runs on my server too.

**Comment by lhbt** [comment]
Hi :) Yes, i am. Tried the same install process twice (the one you advised on the phpbb forum thread) and got the same error twice.

**Comment by lhbt** [comment]
Hmmm, downloading the zip on your releases extracts to a folder called readOthersTopics-1.0.1.
What should the extract folder structure be within phpbb's /ext/ folder ?

I just tried again, same error. This is my folder setup: 
https://gyazo.com/4d0119568b84c22f8202b41e3ef22915

Cheers

**Comment by brunoais** [comment]
create the dirs:
`/brunoais/readOthersTopics/`
Put its contents there.

**Comment by lhbt** [comment]
Still no luck. 

However If I take the whole if block starting at line 334 out of the file, the server stops 500'ing and I can access the forum again.

**Comment by brunoais** [comment]
Are you sure you are using php version 5.6?

**Comment by lhbt** [comment]
Yes: https://gyazo.com/98518db1f56cd36442112f1431b7dfd8

Are you able to get it running on exactly the same setup ?

**Comment by lhbt** [comment]
Just to let you know, I think I've found a fix:

replacing line 334
`if($post_access !== $this->accesses::FULL_READ){`
with

```
$accesses = $this->accesses;
if($post_access !== $accesses::FULL_READ){
```

Seems to do the trick. The server is running and I don't see any breaking impact that syntax change could have.
Cheers

**Comment by brunoais** [comment]
It was as I suspected. Thanks. I'll fix it when I'm back to my PC

**Comment by lhbt** [comment]
No worries. Thank you for your hard work on the extension!

**Comment by brunoais** [comment]
fixed with `534e65d`
